openapi-templates/configuration.mustache

```diff
@@ -602,7 +602,7 @@ def _get_google_auth_token(url='https://sample-metadata-api-mnrpw3mdza-ts.a.run.
 
     # try to see if we're on a VM
     google_identity_message = ''
-    gurl = f'http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/identity?audience={url}'
+    gurl = f'http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/identity?format=full&audience={url}'
```

Comment:
Can we add a comment here to explain the rationale for the `format=full` flag?
(I'm sure our future selves will be thankful.)

Reply:
Yep, done!